src/test/codegen/loads.rs

```diff
@@ -46,7 +46,7 @@ pub fn small_array_alignment(x: [i8; 4]) -> [i8; 4] {
 // dependent alignment
 #[no_mangle]
 pub fn small_struct_alignment(x: Bytes) -> Bytes {
-// CHECK: [[VAR:%[0-9]+]] = load i32* %{{.*}}, align 1
+// CHECK: [[VAR:%[0-9]+]] = load i32, i32* %{{.*}}, align 1
 // CHECK: ret i32 [[VAR]]
     x
```

Comment:
LLVM is switching to opaque pointers, so instructions working with pointers have to supply both the pointer type and the pointee in the interim.
The final result will have only the pointee type, the pointer type always being `ptr` or something like that.